apps/worker/src/app/workflow/usecases/send-message/send-message-push.usecase.ts

```diff
@@ -134,10 +134,22 @@ export class SendMessagePush extends SendMessageBase {
     for (const channel of pushChannels) {
       const { deviceTokens } = channel.credentials || {};
 
-      const [isChannelMissingDeviceTokens, integration] = await Promise.all([
-        this.isChannelMissingDeviceTokens(channel, command),
-        this.getSubscriberIntegration(channel, command),
-      ]);
+      let isChannelMissingDeviceTokens;
+      let integration;
+      try {
+        [isChannelMissingDeviceTokens, integration] = await Promise.all([
+          this.isChannelMissingDeviceTokens(channel, command),
+          this.getSubscriberIntegration(channel, command),
+        ]);
+      } catch (error) {
+        integrationsWithErrors += 1;
+        Logger.error(
+          { jobId: command.jobId },
+          [`Error processing channel for jobId ${command.jobId}`, error.message || error.toString()].join(' '),
+          LOG_CONTEXT
+        );
```

Comment:
suggestion: it was hard for me to read the join 😅

```suggestion
        Logger.error(
          { jobId: command.jobId },
          `Error processing channel for jobId ${command.jobId}. ${error.message || error.toString()}`,
          LOG_CONTEXT
        );
```

Reply:
Done